src/python/WMCore/WMSpec/StdSpecs/StdBase.py

```diff
@@ -1052,7 +1053,7 @@ def getWorkloadCreateArgs():
                      "RunNumber": {"default": 0, "type": int},
                      "RobustMerge": {"default": True, "type": strToBool},
                      "Comments": {"default": ""},
-                     "SubRequestType": {"default": ""},  # used only(?) for RelVals
+                     "SubRequestType": {"default": "", "validate": subRequestType},
```

Comment:
Are we sure if T0 does not need to validate the subrequest type it wont break here?

Reply:
This validation will be triggered for every single workflow creation. But the default value will ensure it's backwards compatible with those not using it.